src/Raven.Server/Documents/PeriodicBackup/BackupTask.cs

```diff
@@ -242,6 +249,15 @@ public async Task<IOperationResult> RunPeriodicBackup(Action<IOperationProgress>
             }
         }
 
+        private DatabaseRecord GetDatabaseRecord()
+        {
+            using (_serverStore.ContextPool.AllocateOperationContext(out TransactionOperationContext context))
+            using (context.OpenReadTransaction())
+            {
+                return _serverStore.Cluster.ReadDatabase(context, _database.Name);
```

Comment:
Probably better to use ReadDatabaseRaw and just get the `EtagForBackup` value. Rename the method to `GetDatabaseEtagForBackup`